coderd/coderd_test.go

```diff
@@ -78,8 +78,13 @@ func TestDERP(t *testing.T) {
 		DERPMap:   derpMap,
 	})
 	require.NoError(t, err)
+	w2Ready := make(chan struct{}, 1)
 	w1.SetNodeCallback(func(node *tailnet.Node) {
 		w2.UpdateNodes([]*tailnet.Node{node})
+		select {
+		case w2Ready <- struct{}{}:
+		default:
+		}
```

Comment:
Since this is only used once it might be cleaner to just close the channel instead of sending to it.

Reply:
I did it this way to avoid double close of the channel since this will get called multiple times, but I can wrap it in a `sync.Once` instead.